.github/workflows/test.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       uses: actions/checkout@main
 
     - name: update jupyter dependencies with repo2docker
-      uses: jupyterhub/repo2docker-action@master
+      uses: yuvipanda/repo2docker-action@docker-push
```

Comment:
For this test workflow, since we also want to push the image with the `test` tag, I guess we can also remove the login to quay and docker push event and just remove the "NO_PUSH" env variable, right?